src/detail/PlayerDetail.js

```diff
@@ -17,23 +17,45 @@ export default class PlayerDetail extends Component {
       this.setState({ player : player });
     }
   }
+
+  handleDelete = async () => {
+    const { history } = this.props;
+    const { player } = this.state;
+    const confirmation = `Are you sure you want to delete ${player.name}?`;
+
+    if (!window.confirm(confirmation)) { return; }
+
+    try {
+      this.setState({ loading : true });
+      await deletePlayer(player.id);
+      history.push('/players');
+
+    }
+    catch (err){
+      this.setState({ loading : false });
+      console.log(err.message);
+    }
+  }
  
   render() {
-    const { player } = this.state;
+    const { player, loading } = this.state;
    
     if (!player) return null;
-    console.log(player.name);
    
     return (
       <div className="PlayerDetail">
-        <h2> {player.name} </h2>
-        <img src={player.url_image} alt={player.name}></img>
-        <h4> {player.position}</h4>
-        <h4> Enrolled: {player.yearEnrolled}</h4> 
-        { player.isTransfer && <h4 className="transfer"> Transfer </h4>}
-        { !player.isTransfer && <h4 className="signee"> Signee </h4>}
-        { player.isActive && <h4 className="active"> Active</h4>}
-        { !player.isActive && <h4 className="graduated"> Graduated </h4>}
+        { loading && <div className="loading"></div>}
+        <div className="ContainerDiv">
+          <h2> {player.name} </h2>
+          <img src={player.url_image} alt={player.name}></img>
+          <h4> {player.position}</h4>
+          <h4> Enrolled: {player.yearEnrolled}</h4> 
+          { player.isTransfer && <h4 className="transfer"> Transfer </h4>}
```

Comment:
since this is a boolean, you could: 
```
{ player.isTransfer && <h4 className="transfer"> Transfer : player.isTranfer.toString()</h4>}
```
I think this will work to stringify your boolean.

Reply:
same goes for the rest of the booleans.